src/Symfony/Bundle/FrameworkBundle/DependencyInjection/FrameworkExtension.php

```diff
@@ -38,8 +38,8 @@ class FrameworkExtension extends Extension
     /**
      * Responds to the app.config configuration parameter.
      *
-     * @param array            $configs
-     * @param ContainerBuilder $container
+     * @param  array            $configs
+     * @param  ContainerBuilder $container
```

Comment:
I suggest separating the param and the throws sections rather than aligning them